modules/rest-api/pages/load-trusted-cas.adoc

```diff
@@ -15,9 +15,11 @@ POST /node/controller/loadTrustedCAs
 [#description]
 == Description
 
-Loads trusted CA (or _root_) certificates that reside on the accessed node into the Couchbase-Server cluster.
-All loaded CA certificates can be accessed by all nodes.
-Loaded certificates can be used to provide authority to the cluster's nodes, and can be used to authenticate clients' access-attempts.
+Loads trusted certificates that reside on the accessed node into the Couchbase-Server cluster.
```

Comment:
Maybe "in the trust store of the couchbase-server" ?

Reply:
I've written: Loads trusted certificates into the Couchbase-Server trust store.